src/js/views/createCertificationAuthority/View.jsx

```diff
@@ -44,11 +45,12 @@ const CreateCertificationAuthority = () => {
   };
 
   return (
-    <ViewContainer headerTitle={t('title')}>
+    <ViewContainer headerTitle={t('createCertificateAuthority:title')}>
```

Comment:
1. Its not `createCertificateAuthority` but `createCertificationAuthority`
2. You don't need to put the `createCertificationAuthority` on every translation text, since the namespace `createCertificationAuthority` is in the first index of the useTranslation hook (the first index is the default namespace for the T function).

This typo broke some translations in the CA creation page:

![image](https://user-images.githubusercontent.com/36794274/150197331-25883baa-3d3a-4164-84d4-66569398e996.png)